compliant_social_django/backends/compliant_facebook.py

```diff
@@ -0,0 +1,8 @@
+from social_core.backends.facebook import FacebookOAuth2
+from ..storage import AuditLogger
+from ..decorators import create_audit_logs
+
+
+@create_audit_logs(AuditLogger())
```

Comment:
Might be neater to initialise the AuditLogger inside the function, not sure though, just seen it done that way typically.